options/options.js

```diff
@@ -0,0 +1,48 @@
+// Saves options to chrome.storage.sync.
+function save_options() {
+  var MB = document.getElementById('MB').value;
```

Comment:
A cast to integer is also needed here to ensure that we actually save a number and not a string, as per the first comment.